### ISSUE TITLE
Make a `typeof` check with `math` methods

### ISSUE THREAD

**Issue by arnavthorat78** [arnavthorat78/Coding-Help-Module#8]
Can you possibly make a `typeof` check with `math` methods?

The reason is that when I run the code below, it gives me a weird output.
```js
const codingHelp = require("coding-help");

console.log(codingHelp.add("28", 1, 2));
```
The output is a string.
```
02812
```

Perhaps, you can add a `typeof` check before adding the numbers.

**Comment by arnavthorat78** [comment]
Yes, I will do that when I release _v.1.3.0_. Thanks for pointing that out!

**Comment by viradex** [comment]
I will actually probably add that in _v1.3.5_ instead.
